odm2api/ODM2/services/readService.py

```diff
@@ -543,30 +543,25 @@ def getPeople(self, ids=None, firstname=None, lastname=None):
             return None
 
     def getAffiliations(self, ids=None, personfirst=None, personlast=None, orgcode=None):
-        """
-        Retrieve a list of ODM2 Affiliation objects.
-
-        Parameters
-        ----------
-        ids: list, default None
-            List of AffiliationIDs
-        personfirst: str, default None
-            Person First Name
-        personlast: str, default None
-            Person Last Name
-        orgcode: str, default None
-            Organization Code
-
-        Returns
-        -------
-        List of Affiliation objects
-
-        Examples
-        --------
-        >>> read.getAffiliations(ids=[39,40])
-        >>> read.getAffiliations(personfirst='Anthony',
-        ...                      personlast='Aufdenkampe')
-        >>> read.getAffiliations(orgcode='LimnoTech')
+        """Retrieve a list of Affiliation objects.
+
+        If no arguments are passed to the function, or their values are None,
+        all Affiliation objects in the database will be returned.
+
+        Args:
+            ids (:obj:`list`, optional): List of AffiliationIDs. Defaults to None.
```

Comment:
@ocefpaf Is the `:obj:` necessary?

Reply:
It will link to the standard lib docs on that object. Hover over/clock the `bool` [here](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/sphinxcontrib.napoleon.html#sphinxcontrib.napoleon.Config.napoleon_google_docstring) and you'll see that in action. But no, we don't really need that.